src/index.js

```diff
@@ -267,6 +271,7 @@ const generateCriticalCssWrapped = async function generateCriticalCssWrapped (
 const m = (module.exports = function (options, callback) {
   // init logging and debug output
   normalizeCss.DEBUG = m.DEBUG
+  const PUPPETEER_EXECUTABLE_PATH = m.PUPPETEER_EXECUTABLE_PATH
```

Comment:
if you declare a const here like this, it will not be accessible in the `launchBrowserIfNeeded` function.
Can you for now just declare this variable as a `let` on line 29 (without any assignment, i.e. `let PUPPETEER_EXECUTABLE_PATH`), to make it accessible in the scope of `launchBrowserIfNeeded`, then on this line just assign the value (`PUPPETEER_EXECUTABLE_PATH = ....`).

Then should work for now, and I will refactor this code a bit in the future.